server/cmd/mmctl/commands/post.go

```diff
@@ -64,31 +65,56 @@ func init() {
 }
 
 func postCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
-	message, _ := cmd.Flags().GetString("message")
```

Comment:
I'm a bit hesitant to remove the `--message` flag as that is a breaking change. Can we support both, the `--message` flag and an optional argument?

Reply:
I don't yet know how to make a command with optional argumements but I will find out how to do it and apply

Reply:
https://pkg.go.dev/github.com/spf13/cobra#MinimumNArgs is what you are looking for.

Reply:
Okay thanks sir